openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -144,7 +144,7 @@ <h3 class="hd hd-6">${_("Pursue a verified certificate")}</h3>
                         <img src="https://courses.edx.org/static/images/edx-verified-mini-cert.png" alt="">
                         <div class="upgrade-container">
                             <p>
-                                <a class="btn-brand btn-upgrade"
+                                <a id="green_upgrade" class="btn-brand btn-upgrade"
```

Comment:
most of the upgrade buttons are green? this may be a bit confusing

Reply:
Yes! (Sorry for the confusion- I should have probably also linked a doc with the categories, especially confusing for a PR grouped by page, not category). Most of the various upsell links fall into one of several categories: FBE banner links, PLS (personalized learner schedule) links, the big green buttons, the welcome coupon, etc. These are to help classify the links and give analytics has the option of comparing links/buttons from the same category. We used the same id or class name for the various links in the same category. This id of "green_upgrade" is used accordingly, as it's in the category of the big green upsell buttons.

Reply:
wouldn't there be multiple big green buttons on course home since the course sock has one and it is also present on course home? this would be a problem since ids need to be unique

Reply:
Yes absolutely- I used id="sock" for those links to avoid the collision. Sorry for the confusion- most of the links were the only one of their kind on the page and could therefore use the category as the id, but this was an exception